arex-schedule-web-api/src/main/java/com/arextest/schedule/sender/impl/DefaultDubboReplaySender.java

```diff
@@ -181,9 +182,54 @@ private DubboParameters getDubboParameters(ReplayActionCaseItem caseItem) {
     DubboParameters dubboParameters = new DubboParameters();
     dubboParameters.setParameterTypes(toParameterTypes(type));
     dubboParameters.setParameters(toParameters(body, type));
+    annotateParameterDataWithClass(dubboParameters, getRecordRequestGenericParameterType(caseItem));
     return dubboParameters;
   }
 
+  private String getRecordRequestGenericParameterType(ReplayActionCaseItem caseItem) {
+    if (caseItem == null || caseItem.getTargetRequest() == null) {
+      return null;
+    }
+    String recordRequestType = (String) caseItem.getTargetRequest().getAttribute("recordRequestType");
+    if (recordRequestType == null) {
+      return null;
+    }
+    recordRequestType = recordRequestType.replaceAll("[\"\\]]+$", "");
+    String[] parts = recordRequestType.split("-");
+    return parts.length > 0 ? parts[parts.length - 1] : null;
```

Comment:
“recordRequestType” may have more than one and should needs to correspond to origin parameterTypes.